recipes-graphics/wayland/weston-init.bbappend

```diff
@@ -6,15 +6,39 @@ IMX_REQUIRED_DISTRO_FEATURES_REMOVE_imxgpu2d = "opengl"
 IMX_REQUIRED_DISTRO_FEATURES_REMOVE_imxgpu3d = ""
 REQUIRED_DISTRO_FEATURES_remove = "${IMX_REQUIRED_DISTRO_FEATURES_REMOVE}"
 
-SRC_URI += "${@bb.utils.contains('DISTRO_FEATURES', 'systemd wayland x11', 'file://weston.config', '', d)}"
+SRC_URI_append_mx6sl = "file://weston.config"
 
-HAS_SYSTEMD = "${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'true', 'false', d)}"
-HAS_XWAYLAND = "${@bb.utils.contains('DISTRO_FEATURES', 'wayland x11', 'true', 'false', d)}"
+# To customize weston.ini, start by setting the desired assignment in weston.ini,
+# commented out. For example:
+#     #xwayland=true
+# Then add the assignment to INI_UNCOMMENT_ASSIGNMENTS.
+INI_UNCOMMENT_ASSIGNMENTS = " \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'x11 wayland', 'xwayland=true', '', d)} \
+"
+INI_UNCOMMENT_ASSIGNMENTS_append_mx7ulp = " \
+    use-g2d=1 \
+"
+INI_UNCOMMENT_ASSIGNMENTS_append_mx8mm = " \
+    use-g2d=1 \
```

Comment:
For imx8mm-evk, I would propose to include following lines as well:
```
\\[output\\] \
name=HDMI-A-1 \
mode=1920x1080@60 \
```
Without them, `fsl-image-multimedia-full` image built with `DISTRO="fslc-wayland"` does not come up with desktop, as the output is not enabled in the weston.ini file.

Reply:
LGTM, i.MX8M Mini EVK has a functional weston.

@thochstein: Once again, thanks a lot for solving this!
--andrey

Reply:
> For imx8mm-evk, I would propose to include following lines as well:
> 
> ```
> \\[output\\] \
> name=HDMI-A-1 \
> mode=1920x1080@60 \
> ```
> 
> Without them, `fsl-image-multimedia-full` image built with `DISTRO="fslc-wayland"` does not come up with desktop, as the output is not enabled in the weston.ini file.

Hi @zandrey, we don't want to hard code a specific display. Without an explicit display configuration, the default behavior should be to detect the active display and use it. Perhaps you're having a problem there.